### PR TITLE
Added Bulgarian language

### PR DESCRIPTION
**Pull request by SangeloSlime**
BG_bg.properties

### CODE REVIEW

**Comment by GamingCrafthd** [comment]
You missed line 66, please translate the line.

**Comment by SangeloSlime** [comment]
oh oops ok

**Comment by GamingCrafthd** [comment]
dont delete it, just edit the file directly in github 🤦🏻‍♂️

File: bot/BG_bg.properties [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+stat_text_channel_value=%text_channel%
+stat_voice_channel_value=%voice_channel%
+stat_categories_value=%categories%
+stat_total_channel_value=%channel%
+stat_normal_emojis_value=%normal_emojis%
+stat_animated_emojis_value=%animated_emojis%
+stat_total_emojis_value=%emojis%
+stat_boosts_value=%boosts%
+stat_boost_level_value=%boost_level%
+stat_guild_age_value=%age% Days
```
Comment (GamingCrafthd) [inline bot/BG_bg.properties]: Days seems english to me xD

**Comment by SangeloSlime** [comment]
oh whoops didnt see that comment

**Comment by SangeloSlime** [comment]
now?

**Comment by GamingCrafthd** [comment]
Approved! Thanks for contributing to Statify! Every commit helps to make statify better!

**Comment by SangeloSlime** [comment]
:)

**Pull request merged**